includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -421,4 +417,96 @@ private function getPartnerSlot() {
 		}
 		return null;
 	}
+
+	private function getLogo() {
```

Comment:
using PHP doc you can also document what is being returned so your IDE can help you out.